R/importNGWMN_wml2.R

```diff
@@ -32,7 +32,7 @@
 #' 
 #' 
 #TODO: separate id and agency name, give also as separate dimensions
-importNGWMN_wml2 <- function(input, asDateTime=FALSE, tz=""){
+importNGWMN_wml2 <- function(input, asDateTime=FALSE, tz){
```

Comment:
Mm yeah.  I dropped the default since there is already a default coming in from readNGWMNdata, and didn't want to set a default at multiple levels (that was what my question in r-activitites was about).